studio/schemas/documents/certificationsSection.js

```diff
@@ -0,0 +1,13 @@
+export default {
+    name: 'certificationsSection',
+    title: 'Certifications Section',
+    type: 'document',
+    fields: [
+        {
+            name: 'certifications',
+            title: 'Certifications',
+            type: 'array',
+            of: [{ type: 'reference', to: { type: 'certification' } }],
+        },
+    ],
+};
```

Comment:
AI don't think you'll need this file. We can just query all of the certifications in the page query